app.js

```diff
@@ -1,12 +1,13 @@
 const express = require("express");
-const { getTopics } = require("./controllers/topics.controller");
+const { getTopics, getEndpoints } = require("./controllers/topics.controller");
 const { invalidPath } = require("./controllers/errors.controller");
 const app = express();
 
 //middleware connections
 app.use(express.json());
```

Comment:
this is from a previous ticket but can you take it out until we're actually using it, think about what endpoint you'll need to add it back in